packages/tea-map-cache/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+const AWS = require('aws-sdk');
+const { getTeaBucketPath } = require('./tea');
+
+/**
+ * Lambda handler that takes a bucketlist and a S3URI to write and
+ * queries TEA for a bucket mapping object.   Returns the bucket map object
```

Comment:
This description doesn't seem correct wrt the params.

Reply:
0fa1eba5f